addons/mail_plugin/controllers/mail_plugin.py

```diff
@@ -184,11 +184,10 @@ def res_partners_search(self, search_term, limit=30, **kwargs):
         search on.
         The method returns an array containing the dicts of the matched contacts.
         """
-
         normalized_email = tools.email_normalize(search_term)
 
         if normalized_email:
-            filter_domain = [('email_normalized', '=', search_term)]
+            filter_domain = [('email_normalized', 'ilike', search_term)]
```

Comment:
Not sure to understand the commit message (talks about URL ?).

Reply:
oops, I mean email, fixed